tokens/.style-dictionary/config.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const { inheritedVariablePrefix } = require( '../build/wikimedia-ui-base/config' ),
+	StyleDictionary = require( 'style-dictionary' ).extend( {
+		include: [ 'node_modules/wikimedia-ui-base/tokens.json' ],
+		source: [ 'properties/**/*.json' ],
+		platforms: {
+			scss: {
+				transformGroup: 'scss',
+				buildPath: 'dist/',
+				files: [ {
+					destination: '_variables.scss',
+					format: 'scss/variables',
+					filter: 'removePrefixedVars',
+				} ],
+			},
+			css: {
+				transformGroup: 'css',
+				buildPath: 'dist/',
+				files: [ {
+					destination: 'variables.css',
+					format: 'css/variables',
+					filter: 'removePrefixedVars',
+				} ],
+			},
+			less: {
+				transformGroup: 'less',
+				buildPath: 'dist/',
+				files: [ {
+					destination: '_variables.less',
+					format: 'less/variables',
+					filter: 'removePrefixedVars',
+				} ],
+			},
+			json: {
+				transformGroup: 'web',
+				buildPath: 'dist/',
+				files: [ {
+					destination: 'index.json',
+					format: 'json',
+					filter: 'removePrefixedVars',
```

Comment:
It appears([0]) we can even use a pointer to a named function so no need for magic strings.

[0]: https://amzn.github.io/style-dictionary/#/formats?id=filtering-tokens

Reply:
Oh great! Changed it and also moved it to a separate file.